src/common/snippets/include/snippets/op/subgraph.hpp

```diff
@@ -143,28 +139,33 @@ class Subgraph : public ov::op::util::SubGraphOp {
     // Return estimated unique buffer count (upper bound). It's needed for tokenization
     static auto get_estimated_buffer_count(const ov::NodeVector& ops) -> size_t;
     static auto is_domain_sensitive_op(const std::shared_ptr<ov::Node>& op) -> bool;
+
+    void data_flow_shape_agnostic(const BlockedShapeVector& blocked_input_shapes = {},
```

Comment:
Here we emphasize that the transformations are performed on data_flow IR. Since we already have `control_flow_transformations` I would rather go with `data_flow_transformations`. How about this?

Reply:
Yeah, I agree. Let's leave it so at least

Reply:
Ok, renamed